src/integration-test/java/mil/dds/anet/integrationTest/emails/AnetEmailWorkerIT.java

```diff
@@ -0,0 +1,112 @@
+package mil.dds.anet.integrationTest.emails;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.beans.AnetEmail;
+import mil.dds.anet.config.AnetConfiguration;
+import mil.dds.anet.config.AnetConfiguration.SmtpConfiguration;
+import mil.dds.anet.database.EmailDao;
+import mil.dds.anet.integrationTest.utils.EmailResponse;
+import mil.dds.anet.integrationTest.utils.FakeSmtpServer;
+import mil.dds.anet.threads.AnetEmailWorker;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({EmailDao.class, AnetConfiguration.class, ScheduledExecutorService.class,
+    AnetObjectEngine.class})
+public class AnetEmailWorkerIT {
+
+  private AnetEmailWorker emailWorker;
+  private EmailDao emailDao;
+  private FakeSmtpServer emailServer;
+  private AnetObjectEngine instance;
+
+  /**
+   * Sets up the test.
+   */
+  @Before
+  public void setUp() {
+    emailDao = PowerMockito.mock(EmailDao.class, Mockito.RETURNS_MOCKS);
+
+    final AnetConfiguration config =
+        PowerMockito.mock(AnetConfiguration.class, Mockito.RETURNS_MOCKS);
+    final ScheduledExecutorService scheduler =
+        PowerMockito.mock(ScheduledExecutorService.class, Mockito.RETURNS_MOCKS);
+
+    // Configuration
+    when(config.getDictionaryEntry("SUPPORT_EMAIL_ADDR")).thenReturn("support@example.com");
+    when(config.getDictionaryEntry("dateFormats.email.date")).thenReturn("d MMMM yyyy");
+    when(config.getDictionaryEntry("dateFormats.email.withTime")).thenReturn("d MMMM yyyy @ HH:mm");
+    when(config.getDictionaryEntry("engagementsIncludeTimeAndDuration")).thenReturn(true);
+    when(config.getDictionaryEntry("activeDomainNames")).thenReturn(Arrays.asList("anet.com"));
+    when(config.getDictionaryEntry("fields")).thenReturn(new HashMap<String, Object>());
+
+    when(config.getEmailFromAddr()).thenReturn("test_from_address@anet.com");
+    when(config.getServerUrl()).thenReturn("localhost:1025");
```

Comment:
Is this needed? And if so, shouldn't it be `localhost:1180` instead (i.e. the HTTP port of the fake SMTP server)?

Reply:
same (and only) question from me.

Reply:
No, it is unnecessary. Removed.